docs/user-guide/Authentication.md

```diff
@@ -30,4 +32,9 @@ Note: in the above example, there are two different applications.  This is only
 
 **Token values in logs**
 
-RESTler has logic to prevent token values from being written to the network logs.  It is recommended to check the RESTler network logs and make sure that the token values are, indeed,  successfully omitted from the logs. 
\ No newline at end of file
+RESTler has logic to prevent token values from being written to the network logs.  It is recommended to check the RESTler network logs and make sure that the token values are, indeed,  successfully omitted from the logs. 
+                
+**Certificate based authentication**
+                
+Certificate and corresponding keys can be used as an authentication machanism. To use this add a key value pair in the settings.json file e.g. 
+                "client_certificate_key_path":"C:\\Users\\<folder></folder>\\privateKey.txt"
```

Comment:
You can just say 'See the SettingsFile.md for the settings that should be used to specify certificates.